test/e2e-tests-yaml.sh

```diff
@@ -29,9 +29,14 @@ for op in apply delete;do
         kubectl ${op} -f ${file}
     done
 
-# Apply docs disabled - pending deletion
-# See https://github.com/tektoncd/triggers/issues/164
-#    for file in $(find ${REPO_ROOT_DIR}/docs -name *.yaml | sort); do
-#        kubectl ${op} -f ${file}
-#    done
 done
+
+
+# Test getting-started examples that use the `getting-started` namespace
+kubectl create namespace getting-started
+for op in apply delete;do
+  for file in $(find ${REPO_ROOT_DIR}/docs -name *.yaml | sort); do
```

Comment:
Can we move [create-ingress.yaml](https://github.com/tektoncd/triggers/blob/master/docs/create-ingress.yaml) and [create-webhook.yaml](https://github.com/tektoncd/triggers/blob/master/docs/create-webhook.yaml) to [getting-started](https://github.com/tektoncd/triggers/tree/master/docs/getting-started) itself as those tasks are used only in getting-started guide so that we can give specifically like below
```suggestion
  for file in $(find ${REPO_ROOT_DIR}/docs/getting-started -name *.yaml | sort); do
```
and we can get rid of yaml files from current docs folder

Reply:
Hmm, sure. I think there was some work to move it to the Catalog too.

Reply:
done!